.github/workflows/test.yml

```diff
@@ -47,9 +43,14 @@ jobs:
             ${{ runner.os }}-mix-${{ github.head_ref || github.ref }}-
             ${{ runner.os }}-mix-refs/heads/master-
 
-
       - run: mix deps.get
       - run: mix format --check-formatted
       - run: mix compile --warnings-as-errors
       - run: mix test
+
+      - run: curl https://clickhouse.com/ | sh
+      - run: sudo ./clickhouse install
```

Comment:
I don't know why, but install takes almost a minute on CI. So I'm moving it and the related tests (just structure_load for now) later in the pipeline to fail fast on all the other tests that don't require clickhouse-client.

Reply:
Maybe there is a way to cache it somehow.

Reply:
we could set the installation folder to something else and cache that folder only

Reply:
How?

Reply:
I can look into caching next after this is merged, should I make a new issue so we don't forget?

Reply:
Sure.

Reply:
> How?

https://github.com/actions/cache

cache with this and then add an `if` to the run option that it should only run if the cache was not hit

Reply:
How can we configure where the install folder?

Reply:
I don't think the current directory matters during install: https://github.com/plausible/ecto_ch/actions/runs/4828815732/jobs/8603098938

Reply:
we could use `./clickhouse install --binary-path Path/To/cache` and then add that path to the system $PATH

Reply:
Nice. I wonder if there is also an option to only install the client.

Reply:
have not seen that option in the output:
```
./clickhouse install --help
Usage: sudo ./clickhouse install [options]
  -h [ --help ]                         produce help message
  --prefix arg (=/)                     prefix for all paths
  --binary-path arg (=usr/bin)          where to install binaries
  --config-path arg (=etc/clickhouse-server)
                                        where to install configs
  --log-path arg (=var/log/clickhouse-server)
                                        where to create log directory
  --data-path arg (=var/lib/clickhouse) directory for data
  --pid-path arg (=var/run/clickhouse-server)
                                        directory for pid file
  --user arg                            clickhouse user to create
  --group arg                           clickhouse group to create
  -y [ --noninteractive ]               run non-interactively
  --link                                create symlink to the binary instead of
                                        copying to binary-path

```